### PR TITLE
ci(tests): new test plan for flaky tests

### PR DESCRIPTION
**Pull request by audie1**
This implements a new test plan which houses the flaky tests, and turns off these tests in the other test plans.

### CODE REVIEW

File: Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme [hunk 1]
```diff
@@ -332,11 +332,6 @@
             value = "pocket-ios-next-dev"
             isEnabled = "YES">
          </EnvironmentVariable>
-         <EnvironmentVariable
```
Comment (bassrock) [inline Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme:335]: Lets not remove this

File: Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme [hunk 2]
```diff
@@ -121,7 +121,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
```
Comment (bassrock) [inline Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme:124]: Lets not change this

Comment (audie1) [reply Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme]: Fixed this

File: Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme (hunk 1 above)
Comment (audie1) [reply Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme]: Fixed this

**Review by Gio2018 — APPROVED** [review]
sounds like a (flaky? 😄 ) plan!

**Comment by audie1** [comment]
I agree @Gio2018 😆 !!

**Comment by audie1** [comment]
Hopefully in time, we will get some of these tests more robust and implement back into the main CI.

**Pull request merged**